### PR TITLE
Add interview with Julia

### PR DESCRIPTION
**Pull request by simonv3**
The first interview with one of our awesome designers! 

Hey @jsimplicio, here's that interview with you finally :)

Let me know if you want me to throw in some other images, I just did some perusing of your work! I also wasn't sure about your preferred handle, or how you'd like to be referred (guess I should have asked), so I just went with "julia", which is what's on twitter.

It might be good to have some sort of header image. Thoughts?

https://github.com/opensourcedesign/organization/issues/28

### CODE REVIEW

File: _posts/2016-04-04-interview-with-julia.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+I'm currently looking for a job designing and developing applications -- even better if in open source :)
+
+## What projects do you contribute to?
+
+I've been contributing to [ScreenCat](https://github.com/maxogden/screencat), a Mac OS application for screensharing built using the electron framework so it's all HTML, CSS, and JS!
+I've been making some crop icons for [openfarm](https://openfarm.cc/en), helping with UI and UX for [QtPass](https://qtpass.org), a cross platform GUI for pass, the standard unix password manager, and finally, I'm a new member of [Hoodie](http://hood.ie) a friendly, complete backend for front-end developers to build their apps on.
+
+![screencat](/images/interviews/julia/screencat.png)
+
+Why do you work on open source projects?
```
Comment (rdbartlett) [inline _posts/2016-04-04-interview-with-julia.md]: Add `##` :smile_cat:

Comment (jancborchardt) [inline _posts/2016-04-04-interview-with-julia.md]: @simonv3 @jsimplicio do you want to finish that up so we can publish? :)

**Comment by simonv3** [comment]
@jancborchardt updated.

**Comment by gillisig** [comment]
Ok I decided to push the button and merge this finally since it has been 1 year and 1 day since our last post and this post has waited way too long! Whoever is in charge of the twitter account (@jancborchardt I think) should definitely spread the word and bring some attention to it.

**Comment by simonv3** [comment]
Thanks @gillisig!

**Comment by simonv3** [comment]
@gillisig wanna be our second interviewee?

**Comment by gillisig** [comment]
Sure, I'll try to find the time to write it down sometime soon.

**Comment by simonv3** [comment]
:D

**Comment by jancborchardt** [comment]
@gillisig I also added you to the Twitter account via https://tweetdeck.twitter.com :)

**Pull request merged**